src/main.c

```diff
@@ -495,7 +495,7 @@ static void handle_sigusr2(int sock, short flags, void *arg)
 
 static void handle_sighup(int sock, short flags, void *arg)
 {
-	log_info("got SIGHUP, re-reading config");
+	log_info("got SIGHUP, re-reading config file %s", cf_config_file);
```

Comment:
I don't think this is appropriate. The message refers to reloading the whole configuration, not only a particular file.